Report/Chapters/Process.tex

```diff
@@ -46,6 +46,8 @@ \section{Process tools}
 
 \subsection{Trello}
 
+The layout of a Trello project with horizontally placed containers which hold vertically aligned cards makes it a good tool for maintaining sprints. Trello also supports check-off lists, assigning people to cards and a built-in calendar with iCal export so important events can show up in everyones agenda. \index{Trello}
```

Comment:
This doesn't really explain what Trello is if you're unfamiliar with it, I think.